### ISSUE TITLE
AttributeError: 'PeriodicTask' object has no attribute '_default_manager'

### ISSUE THREAD

**Issue by idesignedme** [celery/django-celery#472]
I have an app that in deployment on heroku that gives me this error. So I can't run tasks automatically. the error is in 

/app/.heroku/python/lib/python3.5/site-packages/djcelery/schedulers.py at line 98. Locally I fixed this problem by changing this

```
obj = self.model._default_manager.get(pk=self.model.pk)
```

to this

```
    Model = type(self.model)
    obj = Model._default_manager.get(pk=self.model.pk)
```

Im so new to programming I didn't realize that this does not get git pushed. this is local. Will this be fixed soon? or is there a way to edit the heroku package? I ran heroku run bash.  but their file structure is different and I don't want to break my app. How can I fix this?

**Comment by AlkindiX** [comment]
Same problem like me

**Comment by idesignedme** [comment]
@AlkindiX did you get yours to work? mine still doesnt

**Comment by vytisb** [comment]
This issue has been fixed in master branch in 803234d03c8f03a149476365496263a921714bba.

**Comment by AlkindiX** [comment]
I though it did not fixed

On Oct 7, 2016, at 1:24 AM, idesignedme <notifications@github.com<mailto:notifications@github.com>> wrote:


@AlkindiX<https://github.com/AlkindiX> did you get yours to work? mine still doesnt

-
